File: FootScout-Vue.WebAPI/Models/DTOs/RegisterDTO.cs
using System.ComponentModel.DataAnnotations;

namespace FootScout_Vue.WebAPI.Models.DTOs
{
    public class RegisterDTO
    {
        [Required]
        public string Email { get; set; }

        [Required]
        public string Password { get; set; }

        [Required]
        public string ConfirmPassword { get; set; }

        [Required]
        public string FirstName { get; set; }

        [Required]
        public string LastName { get; set; }

        [Required]
        public string PhoneNumber { get; set; }

        [Required]
        public string Location { get; set; }
    }
}

File: FootScout-Vue.WebClient/src/models/dtos/RegisterDTO.ts
// Definiuje dane wymagane do rejestracji użytkownika, w tym e-mail, hasło, imię, nazwisko, 
// numer telefonu i lokalizację.

export interface RegisterDTO {
    email: string;
    password: string;
    confirmPassword: string;
    firstName: string;
    lastName: string;
    phoneNumber: string;
    location: string;
}